.github/workflows/docker.yml

```diff
@@ -1,29 +1,27 @@
 name: Build & Push
-# Build & Push builds the simapp docker image on every push to main and
-# and pushes the image to https://hub.docker.com/r/interchainio/simapp/tags
+# Build & Push rebuilds the cosmos sdk development environment image every time the master branch is updated,
+# and pushes the image to https://ghcr.io/cosmos/cosmos-sdk
+# Other images in the cosmos-sdk consume ghcr.io/cosmos/cosmos-sdk
+
 on:
+  pull_request:
+  schedule:
+    # Run hourly, to speed downstream builds
+    - cron:  '0 * * * *'
   push:
     branches:
       - main
```

Comment:
why hourly if its being updated on every push to main? Seems to be duplicating the work

Reply:
because CI does the work and this ensures that the containers are always up to date.

I can separate this PR from docker changes though, if needed-- except the container that we build the database in.

I very recommend that we discontinue use of this repo:

https://github.com/cosmos/gorocksdb

It is a fork of an unmaintained library.  Anyone actually using rocksdb would thus feel extreme compile time pain because of that choice.